var/spack/repos/builtin/packages/suntans/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Suntans(MakefilePackage):
+    """The Stanford unstructured-grid, 
+    nonhydrostatic, parallel coastal 
+    ocean model. """
+
+    homepage = "https://github.com/ofringer/suntans"
+    url      = "https://github.com/ofringer/suntans/archive/master.zip"
+
+    maintainers = ['ofringer', 'zyaj', 'mrayson', 'jadelson', 'lhxone']
+
+    version('0.0.1', '9b5d0dee5502a5650e37bf72f64b1124be86c084e7408aa517c214ddb25e36de')
+
+    depends_on('libx11@1.6.3', type = ('build', 'run'))
+    depends_on('openmpi', type = ('build'))
+
+    def edit(self, spec, prefix):
+        env['MPIHOME'] = spec['openmpi'].prefix
+        with working_dir('main'):
+            makefile = FileFilter('Makefile')
+            makefile.filter('XINC=.*', 'XINC= -I{0}'.format(spec['libx11'].prefix.include))
+            makefile.filter('XLIBDIR = .*', 'XLIBDIR = {0}'.format(spec['libx11'].prefix.lib))
+            makefile.filter('INCLUDES = .*', 'INCLUDES = $(PARMETISINCLUDE) $(TRIANGLEINCLUDE) $(NETCDFINCLUDE) $(XINC)')
+
+    def build(self, spec, prefix):
+        build_targets = ['CC = mpicc']
+        with working_dir('examples/cylinder'):
+            make('test')
```

Comment:
Can you separate tests from build? You can have a look at the packages implementing:
```python
def check(self):
   pass
```

Reply:
`make test` is used for test environment. It has been removed.